assets/src/scss/bootstrap-build.scss

```diff
@@ -12,57 +12,50 @@ Tags: light, accessibility-ready
 Text Domain: planet4-master-theme
 */
 
-// Full bootstrap:
-// @import "~bootstrap/scss/bootstrap";
+// This file includes commented-out depedencies to make it easier 
+// to track what we are excluding from the original file:
+// https://github.com/twbs/bootstrap/blob/main/scss/bootstrap.scss
 
-@import "~bootstrap/scss/bootstrap-grid.scss";
-@import "~bootstrap/scss/bootstrap-reboot.scss";
+// Configuration
 
-// Required
-// See: https://getbootstrap.com/docs/4.3/getting-started/theming/#importing
 @import "~bootstrap/scss/functions";
 @import "~bootstrap/scss/variables";
 @import "~bootstrap/scss/mixins";
+@import "~bootstrap/scss/utilities";
 
-// TODO: Add variable overrides for the 16px grid here.
+// Layout & components
+@import "~bootstrap/scss/root";
```

Comment:
We had these first files commented out before, do we now want to use them or is this a mistake? 🤔

Reply:
Yes, we want! These were included before as part of `bootstrap-grid` and `bootstrap-reboot`, which are partials including a subset of things. This was looks closer to the original file and makes it easier to see what's being loaded with more granularity.